start.py

```diff
@@ -679,6 +703,36 @@ def _configure_kudu(deployment, cluster, kudu_version):
             # Only one parcel can be activated at a time, so once one is found, our work is done.
             break
 
+
+def _validate_kudu_version(kudu_version, cdh_version):
+    cdh_version_tuple = tuple([int(i) for i in cdh_version.split('.')])
+    cdh_5_13_0_tuple = 5, 13, 0
```

Comment:
Instead of calling it `cdh_5_13_0` tuple, perhaps call it something indicative of what made CDH 5.13 special and put it at the top of the file? e.g. `EARLIEST_CDH_WITH_KUDU_VERSION`

Reply:
Done.